dev/build_release.py

```diff
@@ -196,26 +122,17 @@ def __init__(self, options):
 
       os.environ['NODE_ENV'] = os.environ.get('NODE_ENV', 'dev')
       self.__build_number = options.build_number
-      self.__google_service_account = options.google_service_account
+      self.__gcb_service_account = options.gcb_service_account
       self.__options = options
+      if (self.__options.container_builder
```

Comment:
Could add another check that `container_builder` is populated when `build_container_image` is true.